test/test.js

```diff
@@ -35,7 +40,10 @@ test('equals to sample', async t => {
 
 test('check internal structure', async t => {
   // Create deck as in previous example
-  const apkg = new AnkiExport('deck-name');
+  const apkg = new AnkiExport('deck-name', {
+    template,
+    DbClass: sql.Database
```

Comment:
invalidate cache via `delete require.cache[require.resolve('./../src/index')]` is good idea?